### PR TITLE
words extraction from image pdf

### PR DESCRIPTION
**Pull request by henryspg**
This code is about words extraction from image pdfs

### CODE REVIEW

**Review by ssbyrne89 — COMMENTED** [review]
Nice README file, describes the project very clearly. try to keep those PDFs from being merged though

File: README - lab30.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+!sudo apt-get install tesseract-ocr
+
+
+
+## Recommendation:
+We need to find out a different model to extract the important keywords.
+Note: a small error in a legal document could cause us a big trouble.
+
+
+
```
Comment (jproctor-rebecca) [inline README - lab30.md]: all of the information for your personal work goes in your colab or the README.md in the folder for the notebooks section

It should not appear in the general project README.md

File: README - lab30.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ *How many procent is the case rejected with certain nationality or gender?
+ *Is there any trend based on religion or social group?
+ *Is certain judge in favor of an applicant from a certain case? or country?
+
+In order to answer those questions, we need to extract those keywords and put them into dataframe, and from there we store them
+in our database.
+
+
+Lab30-note:
+Lab29 has tried to extract the keywords using spacy with small model (en_core_web_sm). In this work we implemented the large model os spacy (en_core_web_lg) with expectation to get a more accurate keywords. We also tried to use "regex" to extract the desired words,  but it turned out that this solution also can not give us the accurate keywords.
```
Comment (jproctor-rebecca) [inline README - lab30.md]: more personal work notes - should go in your colab or in the notebooks README.md

File: README - lab30.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Description
+We built an application for human rights first, a 501(c)3 organization. Our application uses optical character recognition to scan input court decisions for such values as the name of the presiding judge, the decision, and the asylum seeker's country of origin, and inserts these values into a database. The hope is that advocates for asylum seekers can use these data to better tailor their arguments before a particular judge and maximize their client's chances of receiving asylum. Based on our stakeholder meeting HRF wants to know the trend and exploratory of the judge's decision, for example:
+
+ *How many procent is the case rejected with certain nationality or gender?
+ *Is there any trend based on religion or social group?
+ *Is certain judge in favor of an applicant from a certain case? or country?
+
+In order to answer those questions, we need to extract those keywords and put them into dataframe, and from there we store them
+in our database.
```
Comment (jproctor-rebecca) [inline README - lab30.md]: Excellent addition to project README.md because it covers whole group work and speaks to general process

Thank you!

**Review by jproctor-rebecca — CHANGES_REQUESTED** [review]
please delete the pdfs from the PR and review comments

File: README - lab30.md (hunk 2 above)
Comment (henryspg) [reply README - lab30.md]: Yes,  working on it now.

**Review by henryspg — COMMENTED** [review]
Filtered out unneeded files

File: README - lab30.md (hunk 1 above)
Comment (henryspg) [reply README - lab30.md]: personal work was taken out. done.  Thanks.

**Comment by henryspg** [comment]
Readme & code are  in the folder: notebooks

**Pull request closed without merging**